<launch>
  <arg name="rate" default="30" />
  <arg name="type" default="visual_servoing" />
  <arg name="args" default="[]" />

  <node name="control_node" pkg="control" type="control_node.py" output="screen" >
      <param name="rate" value="$(arg rate)"></param>
      <param name="type" value="$(arg type)"></param>
      <rosparam param="args" subst_value="True">$(arg args)</rosparam>
  </node>
</launch>